src/living/helpers/babylonjs/InteractiveDynamicTexture.test.ts

```diff
@@ -0,0 +1,30 @@
+import { InteractiveDynamicTexture } from './InteractiveDynamicTexture';
+import DOMMatrix from '../../geometry/DOMMatrix';
+import { describe, it, expect } from '@jest/globals';
+
+describe('InteractiveDynamicTexture', () => {
+  it('should return correct matrix', () => {
+    const transformStr = 'rotate(90deg) translateX(10px)';
+    const expectedMatrix = new DOMMatrix([
+      0, 1, 0, 0,   
+      -1, 0, 0, 0,  
+      0, 0, 1, 0,  
+      0, 10, 0, 1
+    ]);
+    const result = InteractiveDynamicTexture._parserTransform(transformStr);
```

Comment:
Ditto, `InteractiveDynamicTexture` should not contain a function that does parsing, you have to finish it in CSS parsing stage in this project.